packages/cli/src/oclif/commands/register.js

```diff
@@ -32,7 +138,42 @@ RegisterCommand.args = [
       "Your integrations's public title. Asked interactively if not present.",
   },
 ];
-RegisterCommand.flags = buildFlags();
+
+RegisterCommand.flags = buildFlags({
+  commandFlags: {
+    desc: flags.string({
+      char: 'D',
+      description:
+        'A sentence describing your app in 140 characters or less, e.g. "Trello is a team collaboration tool to organize tasks and keep projects on track."',
+    }),
+    url: flags.string({
+      char: 'u',
+      description:
+        'The homepage URL of your app. This will also be used as the email domain that would allow team members to join your integration as a Collaborator.',
+    }),
+    audience: flags.string({
+      char: 'a',
+      description:
+        'Are you building a public or private integration? Public integration development receives priority support and launch promotion from the Zapier team.',
```

Comment:
I think what we're working on makes the second sentence no longer true, so:

```suggestion
        'Are you building a public or private integration?',
```

Reply:
@eliangcs Good call!